draft-ietf-ice-rfc5245bis.xml

```diff
@@ -1163,111 +1090,66 @@ initiating and the responding agent.
+type (server reflexive, peer reflexive, relayed, and host), a
+preference for IP address for which the candidate was obtained, and
+component ID using the following formula:
 </t>
 
 <figure><artwork>
 <![CDATA[
 priority = (2^24)*(type preference) +
-           (2^8)*(local preference) +
+           (2^8)*(IP address preference) +
            (2^0)*(256 - component ID)
 
 ]]></artwork></figure>
 
 <t>
-The type preference MUST be an integer from 0 to 126 inclusive, and
-represents the preference for the type of the candidate (where the
-types are local, server reflexive, peer reflexive, and relayed). A 126
-is the highest preference, and a 0 is the lowest. Setting the value to
-a 0 means that candidates of this type will only be used as a last
-resort. The type preference MUST be identical for all candidates of
-the same type and MUST be different for candidates of different
-types. The type preference for peer reflexive candidates MUST be
-higher than that of server reflexive candidates. Note that candidates
+The type preference MUST be an integer from 0 (lowest prefereence) to
+126 (highest preference) inclusive and MUST be identical for all
+candidates of the same type and MUST be different for candidates of
+different types. The type preference for peer reflexive candidates
+MUST be higher than that of server reflexive candidates.
+Setting the value to 0 means that candidates of this type will only 
+be used as a last resort. Note that candidates
 gathered based on the procedures of <xref target="sec-gathering"/>
 will never be peer reflexive candidates; candidates of these type are
 learned from the connectivity checks performed by ICE.
 </t>
 
-<t>The local preference MUST be an integer from 0 to 65535
-inclusive. It represents a preference for the particular IP address
-from which the candidate was obtained. 65535 represents the highest
-preference, and a zero, the lowest. When there is only a single IP
-address, this value SHOULD be set to 65535. More generally, if there
+<t>The IP address preference MUST be an integer from 0 (lowest preference) 
+to 65535 (highest preference) inclusive. When there is only a single IP
+address, this value SHOULD be set to 65535.  If there
 are multiple candidates for a particular component for a particular
 media stream that have the same type, the local preference MUST be
-unique for each one. In this specification, this only happens for
-multihomed hosts or if an agent is using multiple TURN servers.  If a
-host is multihomed because it is dual-stack, the local preference
+unique for each one. If an agent is dual-stack, the IP address preference
 should be set according to the current best practice described in <xref
 target="I-D.ietf-ice-dualstack-fairness"/>.</t>
 
 
-<t>The
-component ID is the component ID for the candidate, and MUST be
-between 1 and 256 inclusive.
-</t>
+<t>The component ID MUST be an integer between 1 and 256 inclusive.</t>
```

Comment:
I really don't see what "The component ID is the component ID" adds in clarity.  Do you really want me to put it back?